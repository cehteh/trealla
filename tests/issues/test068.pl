:- initialization(main).

main :-
    prepare(List),
    writeln(List),
    sort(List, ListSorted),
    writeln(ListSorted),
    halt.

prepare([B,A]) :-
    A =.. [pair,2,X],
    B =.. [trio,3,Y,Z].
