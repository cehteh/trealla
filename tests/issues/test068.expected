[trio(3,_7,_8),pair(2,_6)]
[pair(2,_6),trio(3,_7,_8)]
